addons/l10n_au/views/report_payment_receipt_templates.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <template id="report_payment_receipt_document" inherit_id="account.report_payment_receipt_document">
+        <xpath expr="//div[hasclass('page')]/h3/strong" position="replace">
+            <!-- Customary in Australia to use the term "Remittance Advice" for payments to suppliers -->
+            <t t-if="o.partner_type and (o.company_id.account_fiscal_country_id.code == 'AU' or o.partner_id.country_id.code == 'AU')">
+                <strong t-if="o.partner_type == 'customer'">Payment Receipt: <span t-field="o.name"/></strong>
+                <strong t-if="o.partner_type == 'supplier'">Remittance Advice: <span t-field="o.name"/></strong>
```

Comment:
generally, lgtm and tested functionally.
But here, since we only need to change the label if the partner is a supplier, then maybe we could add that in the main condition and remove the line for the customer `partner_type`? What do you think? :slightly_smiling_face:

Reply:
Thank you for the review.
Indeed this can work, and avoid repetition at the cost of a long condition :smile: 
I applied the change